app/views/find/courses/_about_the_provider.html.erb

```diff
@@ -9,13 +9,13 @@
   <% end %>
 
   <% if course.accrediting_provider.present? %>
-    <h3 class="govuk-heading-m">About the accredited provider</h3>
-
     <% if course.about_accrediting_provider.present? %>
-      <div data-qa="course__about_accrediting_provider">
-        <%= markdown(course.about_accrediting_provider) %>
-      </div>
-    <% else %>
+        <h3 class="govuk-heading-m">About the accredited provider</h3>
+        <div data-qa="course__about_accrediting_provider">
+          <%= markdown(course.about_accrediting_provider) %>
+        </div>
+    <% elsif params[:action] == "preview" && !course.about_accrediting_provider.present? %>
```

Comment:
```suggestion
<% elsif params[:action] == "preview" %>
```

i don't think the second part is needed since it's already inverted from the initial check?

Reply:
Good point - have updated